packages/react-data-grid/src/index.js

```diff
@@ -1,12 +1,15 @@
 import Grid from './ReactDataGrid';
 import RowComparer from 'common/utils/RowComparer';
 import Cell from './Cell';
+import Row from './Row';
```

Comment:
`Row` uses [`export default`](https://github.com/adazzle/react-data-grid/blob/am-no-class-assign/packages/react-data-grid/src/Row.js#L205) and `require` syntax returns an object instead of the default import